build.gradle.kts

```diff
@@ -1,8 +1,8 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    id("org.springframework.boot") version "3.2.5"
-    id("io.spring.dependency-management") version "1.1.5"
+    id("org.springframework.boot") version "3.2.8"
```

Comment:
Kan vi ikke bumpe til 3.3.x?

Reply:
Tanken var å gjøre minst mulig forskjell for å fjerne sårbarhet, 3.2.8 og 3.3.2 kom begge 18. juli: https://mvnrepository.com/artifact/org.springframework.boot/spring-boot

Men vi bør absolutt oppgradere til 3.3, enten her eller i egen branch.